nomad/core_sched.go

```diff
@@ -703,3 +705,10 @@ func allocGCEligible(a *structs.Allocation, job *structs.Job, gcTime time.Time,
 
 	return timeDiff > interval.Nanoseconds()
 }
+
+// csiVolumeGC is used to garbage collect CSI volumes
+func (c *CoreScheduler) csiVolumeGC(eval *structs.Evaluation) error {
+	// TODO: implement me!
+	c.logger.Trace("garbage collecting unclaimed CSI volumes")
```

Comment:
When we're done running this, the `CSIVolume` object will still exist, it'll just be unpublished on this node. I think we'd want to say something like "unpublishing CSI volume claims"